forch/forchestrator.py

```diff
@@ -82,6 +86,24 @@ def initialize(self):
         self._cpn_collector.initialize()
         LOGGER.info('Using peer controller %s', self._get_peer_controller_url())
         self._register_handlers()
+
+        r_info = self._config.get('radius_info')
+        if r_info:
+            radius_ip = r_info.get('server_ip')
+            radius_port = r_info.get('server_port')
+            secret = r_info.get('secret')
+            if not (radius_ip and radius_port and secret):
+                LOGGER.warning('Invalid radius_info in config.')
+            else:
+                self._authenticator = Authenticator(radius_ip, radius_port, secret)
```

Comment:
make sure success is logged, either here or inside of Authenticator. Include radius_ip and raduis_port, but not secret

Reply:
done.